molecule/default/converge.yml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
Is converge.yml the preferred name for the Molecule playbook nowadays?

Reply:
yes - `converge.yml` and `test.yml` are the new "playbook". But I'm still using testinfra so no need for `test.yml`